src/Web/ViewModels/Account/RegisterViewModel.cs

```diff
@@ -57,8 +57,8 @@ public class RegisterViewModel
 
```

Comment:
This file and the CheckoutDetailsViewModel look like they share quite a few properties; any reason not to pull out the common into a base model?

Reply:
Some of the other view models seem to share properties as well

Reply:
Fair point. Let me see if I can clean it up.... or maybe in the latest eShopOnWeb project has newer models/hints. I know we're "way" behind it by now.  Kind of a long standing thing I'd like to update, but there's not really much need.

Reply:
Cleaned up!  Please take a look and rejoice :-)